azure/packages/azure-client/src/AzureClient.ts

```diff
@@ -60,14 +64,15 @@ export class AzureClient {
      * @param props - Properties for initializing a new AzureClient instance
      */
     constructor(private readonly props: AzureClientProps) {
+        // remove trailing slash from URL if any
```

Comment:
Why?

Reply:
if a user supplies `https://domain.com/` instead of `https://domain.com` then their app will fail to communicate with the fluid relay service. The driver expects that the URLs are exact and will simply append paths to the endpoint URL string (eg. `+ '/alfred/...` so if we have a trailing slash in the URL then we'd get `https://domain.com//alfred/...`)

I thought that it would be more intuitive to have the client strip away the trailing slash rather than silently continue and let the drive fail

Reply:
I agree, but I've already spoken with the driver peeps about this, and they insisted that the driver implicitly trusts whatever URL is given to them. So as a compromise I added this check on the client side.

Reply:
Ok. Thanks for following up.

Reply:
Thanks for fixing this but but yea this is a bit odd that we are handling the logic here instead of doing proper URL building in the driver. We can keep this for now but let's followup with the driver folks.

Reply:
Driver should be able to handle this sort of scenarios, as they are appending and building the final url. Could you create a ticket around that, and we can resolve this conversation?

Reply:
glad to know I'm not the only one who got stuck on the trailing slash 😄 
Here's the issue for it https://github.com/microsoft/FluidFramework/issues/10422